doc/getting-started.rst

```diff
@@ -17,6 +17,35 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
 
 See :doc:`features/parsing` for additional ways to instantiate a Money object from strings.
 
+Accepted integer values
+-----------------------
+The Money object only supports integer(ish) values on instantiation. The following is (not) supported.
+
+.. code-block:: php
+
+    use Money\Currency;
+    use Money\Money;
+
+    // int is accepted
+    $fiver = new Money(500, new Currency('USD'));
+
+    // string is accepted if integer
+    $fiver = new Money('500', new Currency('USD'));
+
+    // string is accepted if fractional part is zero
+    $fiver = new Money('500.00', new Currency('USD'));
+
+    // leading zero's are accepted
+    $fiver = new Money('00500', new Currency('USD'));
+
+    // multiple zero's is not accepted, throws \InvalidArgumentException
```

Comment:
IMO not accepted is always the same as InvalidArgumentException, so it should be mentioned once somewhere.